src/Data/Parity/HomomorphismNatParity.agda

```diff
@@ -0,0 +1,103 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties about parities
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Parity.HomomorphismNatParity where
+
+open import Data.Nat.Base
+  using (zero; suc; _+_; _*_; parity
+  ; +-rawMagma; +-0-rawMonoid
+  ; +-*-rawNearSemiring; +-*-rawSemiring)
+open import Data.Parity.Base using (Parity; 0ℙ; 1ℙ; _⁻¹)
+  renaming (_+_ to _+ℙ_; _*_ to _*ℙ_
+  ; +-rawMagma to ℙ+-rawMagma; +-0-rawMonoid to ℙ+-0-rawMonoid
+  ; +-*-rawNearSemiring to ℙ+-*-rawNearSemiring; +-*-rawSemiring to ℙ+-*-rawSemiring)
+open import Data.Parity.Properties using (⁻¹-inverts)
+open import Relation.Binary.PropositionalEquality
+  using (_≡_; refl; sym; cong; cong₂; module ≡-Reasoning)
+
+open import Algebra.Morphism.Structures
+  using (IsMagmaHomomorphism; IsMonoidHomomorphism
+  ; IsNearSemiringHomomorphism; IsSemiringHomomorphism)
+
+------------------------------------------------------------------------
+-- relating Nat and Parity
+
+-- successor and (_⁻¹)
+
+suc-homo-⁻¹ : ∀ n → (parity (suc n)) ⁻¹ ≡ parity n
+suc-homo-⁻¹ zero    = refl
+suc-homo-⁻¹ (suc n) = ⁻¹-inverts (suc-homo-⁻¹ n)
+
+-- parity is a _+_ homomorphism
+
++-homo-+      : ∀ m n → parity (m + n) ≡ (parity m) +ℙ (parity n)
```

Comment:
Unnecessary brackets round the `(parity m)`?

Reply:
... and `(parity n)` it turns out. Fixed now.